samples/link-unfurling-stt/nodejs/appPackage/manifest.json

```diff
@@ -0,0 +1,90 @@
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.11/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.11",
+  "version": "1.0.0",
+  "id": "<<Microsoft-App-Id>>",
+  "packageName": "com.microsoft.teams.linkunfurlingstt",
+  "developer": {
+    "name": "Microsoft",
+    "websiteUrl": "https://example.azurewebsites.net",
+    "privacyUrl": "https://example.azurewebsites.net/privacy",
+    "termsOfUseUrl": "https://example.azurewebsites.net/termsofuse"
+  },
+  "icons": {
+    "color": "icon-color.png",
+    "outline": "icon-outline.png"
+  },
```

Comment:
Your actual icons are with different name "color.png"/"outline.png"
Kindly update it properly.